server/src/BGS.Infrastructure/Data/EntityConfigurations/GameConfiguration.cs

```diff
@@ -0,0 +1,21 @@
+﻿using BGS.ApplicationCore.Entities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace BGS.Infrastructure.Data.EntityConfigurations;
+
+public class GameConfiguration : IEntityTypeConfiguration<Game>
+{
+    public void Configure(EntityTypeBuilder<Game> builder)
+    {
+        builder.HasKey(g => g.Id);
+
+        builder.Property(g => g.Id)
+            .HasColumnName("game_id")
+            .IsRequired();
```

Comment:
This is not necessary since by default the primary key is required. Also no need set column name